tools/package-tools/build-release.ts

```diff
@@ -65,7 +65,7 @@ export function composeRelease(buildPackage: BuildPackage) {
     // re-exports everything.
     createMetadataReexportFile(
         releasePath,
-        buildPackage.secondaryEntryPoints.map(p => `./${p}`),
+        buildPackage.secondaryEntryPoints.concat(['typings/index']).map(p => `./${p}`),
```

Comment:
Isn't that already handled a few lines above? 
> https://github.com/jelbourn/material2/blob/31ea051ca341e63d3f2f4b0e5e98316d60c9cf54/tools/package-tools/build-release.ts#L51

Reply:
This call to `createMetadataReexportFile` overwrites the file generated by the earlier call

Reply:
Okay that makes sense. I wonder if it would be more obvious if we place the earlier call in the `else` clause.